new-docs/content/any/project/changelogs/2021-02-17.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Release 2021-02-17
+menuTitle: 2021-02-17
+any: true
+description: Changelog for Release 2021-02-17 (oso 0.11.0) containing new features, bug fixes, and more.
+---
+
+## `oso` 0.11.0
+
+### Unbound variable handling in the Polar VM
+
+The Polar Virtual Machine now handles expressions involving unbound
+variables by adding constraints to them. For example, given a rule like:
+```
+f(x) if x > 1;
+```
+A query for `f(x)` where `x` is an unbound variable will now return
+a constraint represented as the expression `_this > 1`. This is
+intended to be used in data filtering contexts, such as with the
+Django or SQLAlchemy adapters.
+
+Because extensive internal changes were made to support this feature,
+we recommend that you test your policies carefully with this version.
+Please reach out via [Slack](https://join-slack.osohq.com/) or a
+[GitHub issue](https://github.com/osohq/oso/issues) if you find any
+problems, e.g., policies that now produce errors or incorrect results,
+missing constraints in data filtering, etc.
```

Comment:
```suggestion
problems, e.g., policies that now produce errors.
```

And update the data filtering section below to say that data filtering is still considered an experimental feature.

Reply:
(e.g. perf and completeness)